### PR TITLE
Converted camelCase field names to title case.

### PR DESCRIPTION
**Pull request by jonmccormick**
For issue #58

### CODE REVIEW

File: src/listener/utils/transforms/resource.js [hunk 1]
```diff
@@ -28,4 +28,24 @@ export default class Resource {
     this.description = oneViewResource.description;
     this.hyperlink = oneViewResource.hyperlink;
   }
+
+ camelCaseToTitleCase(camelCase){
+    if (camelCase === null || camelCase === "") {
+      return camelCase;
+    }
+    camelCase = camelCase.trim();
+    var newText = "";
```
Comment (danielreed) [inline src/listener/utils/transforms/resource.js]: Instead of var use let

File: src/listener/utils/transforms/resource.js [hunk 2]
```diff
@@ -28,4 +28,24 @@ export default class Resource {
...
+
+    if (/[a-z]/.test(camelCase[0])){
+      newText = camelCase[0].toUpperCase();
+    }
+
+    for (var i = 1; i < camelCase.length; i++) {
```
Comment (danielreed) [inline src/listener/utils/transforms/resource.js]: Use let

File: src/listener/utils/transforms/server-hardware.js [hunk 3]
```diff
@@ -43,8 +43,11 @@ export default class ServerHardware extends Resource {
       if (this.__isNonDisplayField__(field) || !this[field]) {
         continue;
       }
+
+      var displayField = this.camelCaseToTitleCase(field);
```
Comment (danielreed) [inline src/listener/utils/transforms/server-hardware.js]: Use let

File: src/listener/utils/transforms/resource.js (hunk 1 above)
Comment (jonmccormick) [inline src/listener/utils/transforms/resource.js]: Done.

File: src/listener/utils/transforms/resource.js (hunk 2 above)
Comment (jonmccormick) [inline src/listener/utils/transforms/resource.js]: Done

File: src/listener/utils/transforms/server-hardware.js (hunk 3 above)
Comment (jonmccormick) [inline src/listener/utils/transforms/server-hardware.js]: Done

File: src/listener/utils/transforms/server-profile-template.js [hunk 4]
```diff
@@ -38,8 +38,11 @@ export default class ServerProfileTemplate extends Resource {
       if (this.__isNonDisplayField__(field) || !this[field]) {
         continue;
       }
+
+      let displayField = this.amelCaseToTitleCase(field);
```
Comment (danielreed) [inline src/listener/utils/transforms/server-profile-template.js]: needs to be camelCaseToTitleCase I think

Comment (danielreed) [inline src/listener/utils/transforms/server-profile-template.js]: I'll merge this PR now and you can follow up with a new one to fix this issue.

**Pull request merged**